### PR TITLE
Added documentation for only migrating data hub and studio.

### PR DESCRIPTION
**Pull request by MarvGilb**
Signed-off-by: Marvin Gilbert <Marvin.Gilbert@here.com>

### CODE REVIEW

File: docs/letting-go.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+
+## Converting to from Data Hub to IML
```
Comment (hmorch) [inline docs/letting-go.md:2]:
```suggestion
## Converting from Data Hub to IML
```
Remove first 'to'?

File: docs/letting-go.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+You can migrate Data Hub spaces associated with your [developer.here.com](https://developer.here.com) accounts to the [HERE platform](https://platform.here.com) using the self-service Data Hub migration tool.
+
+This section outlines how to migrate your Data Hub spaces to the HERE platform using the self-service Data Hub migration tool:
+
+## Before you migrate
+
+Before you migrate your Data Hub spaces to the HERE Platform, HERE recommends that you perform the following pre-migration steps:
+
+- Check with your team to see if you have an existing HERE platform organization. Any number of developer.here.com accounts can migrate their resources to a single HERE platform organization.
```
Comment (hmorch) [inline docs/letting-go.md:14]:
```suggestion
- Check with your team to see if you have an existing HERE platform organization. Any number of developer.here.com accounts can migrate their spaces to a single HERE platform organization.
```
'resources' sounds very much like the general migration.

File: docs/letting-go.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+
+## Converting to from Data Hub to IML
+
+> Data Hub is currently "[In Maintenance](https://developer.here.com/documentation/product-lifecycle-policy/customer_notice/index.html)". Interactive map layers (IML) has been identified as the successor to Data Hub. For new users considering Data Hub, HERE recommends that you start with IML on the [HERE platform](https://platform.here.com).
```
Comment (hmorch) [inline docs/letting-go.md:4]: Why the '>'  for a quote? What do you quote? Source?

File: docs/letting-go.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## Migrate your resources
+
+To migrate your Data Hub spaces to the HERE platform, create or select a Data Hub token, that has **full** (all spaces) 'manageSpaces' rights for your user.
+
+### Use the following snippet to perform a dryrun:
+````
+curl -X 'GET' 'https://conversion-service-eu-west-1.api-gateway.ls.hereapi.com/list/dh' \
+  -H 'accept: application/json' \
+  -H 'Authorization: Bearer eyPlatformTokenOfDestinationUser' \
```
Comment (hmorch) [inline docs/letting-go.md:40]:
```suggestion
  -H 'Authorization: Bearer <PlatformToken>' \
```
I don't think we should put real tokens on this page. We don't for the data hub token too.

File: docs/letting-go.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  -H 'Dh-Token: data-hub-short-token'
```
Comment (hmorch) [inline docs/letting-go.md:41]:
```suggestion
  -H 'Dh-Token: <data-hub-short-token>' \
```
Should we realy have an upper case 'D' here?
We should use a common delimiter for values the user has to set.

File: docs/letting-go.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+> If your organization has signed up for [HERE Marketplace](https://www.here.com/platform/marketplace), contact your organization admin who can invite you to join the HERE platform organization established for your company. You can also request a free trial of the HERE platform if your company does not have an organization established for it. For more information, see the [HERE platform pricing](https://developer.here.com/pricing/platform).
+>
+> You may need to add a credit card to your HERE platform account as part of the migration process. For example, you must have a credit card associated with your account to migrate HERE Data Hub resources.
+
+To sign up for a HERE platform account, follow these steps:
+
+1. Sign in to your [developer.here.com account](https://developer.here.com/login).
+2. Click **Create a platform account to get started** from the yellow page banner or navigate to [https://platform.here.com/sign-up](https://platform.here.com/sign-up).
+3. Follow the prompts to complete your HERE platform account creation or click **Sign in instead** to sign in to an existing HERE platform account.
+
```
Comment (hmorch) [inline docs/letting-go.md:31]: Please add a section how to copy data manually for testing here. As reference/source you can use https://docs.in.here.com/static/100880/1001824/html/topics/here-data-hub.html

**Review by hmorch — CHANGES_REQUESTED** [review]

File: docs/letting-go.md (hunk 3 above)
Comment (MarvGilb) [reply docs/letting-go.md:4]: Copied it from here: https://github.com/heremaps/xyz-documentation/blob/master/docs/api/README.md

Comment (hmorch) [reply docs/letting-go.md:4]: But your version does not have the "Note" header. That would make more sense.

File: mkdocs.yml [hunk 7]
```diff
@@ -53,6 +53,7 @@ extra:
 nav:
   - Introduction: index.md
   - Getting Started: getting-started.md
+  - Letting Go: letting-go.md
```
Comment (hmorch) [inline mkdocs.yml:56]: "Letting Go" will show up in menu? I don't think that's a good title. "Migration to Platform" or like the title of this page "Converting from Data Hub to IML"

File: docs/letting-go.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+## Converting from Data Hub to IML
```
Comment (hmorch) [inline docs/letting-go.md:2]: This is the page titile? So it should be one header level higher then the others. Currently it's on the same lavel as e.g. "Before you migrate".
Also we are mixing "convert(ing)" and "migrate(ing)". We should stick to one and I prefer migrate.

**Comment by hmorch** [comment]
Outdated.

**Review by hmorch — APPROVED** [review]

**Pull request merged**